src/locale/locales/vi/messages.po

```diff
@@ -0,0 +1,8540 @@
+msgstr "Bạn cần xác minh email trước khi có thể tạo bài viết."
+
+#: src/components/StarterPack/ProfileStarterPacks.tsx:340
+msgid "Before creating a starter pack, you must first verify your email."
+msgstr "Bạn cần xác minh email trước khi có thể tạo gói khởi đầu."
+
+#: src/components/dms/dialogs/NewChatDialog.tsx:79
+#: src/components/dms/MessageProfileButton.tsx:89
+#: src/screens/Messages/Conversation.tsx:219
+msgid "Before you may message another user, you must first verify your email."
+msgstr "Bạn cần xác minh email trước khi có thể nhắn tin với người dùng khác."
+
+#: src/components/dialogs/BirthDateSettings.tsx:106
+#: src/screens/Settings/AccountSettings.tsx:101
+msgid "Birthday"
+msgstr "Ngày sinh"
+
+#: src/view/screens/Settings/index.tsx:348
+#~ msgid "Birthday:"
+#~ msgstr "Ngày sinh:"
+
+#: src/screens/Profile/Header/ProfileHeaderStandard.tsx:283
+#: src/view/com/profile/ProfileMenu.tsx:341
+msgid "Block"
+msgstr "Chặn"
+
+#: src/components/dms/ConvoMenu.tsx:188
+#: src/components/dms/ConvoMenu.tsx:192
+msgid "Block account"
+msgstr "Chặn tài khoản"
+
+#: src/view/com/profile/ProfileMenu.tsx:280
+#: src/view/com/profile/ProfileMenu.tsx:287
+msgid "Block Account"
+msgstr "Chặn tài khoản"
+
+#: src/view/com/profile/ProfileMenu.tsx:324
+msgid "Block Account?"
+msgstr "Chặn tài khoản"
+
+#: src/view/screens/ProfileList.tsx:643
+msgid "Block accounts"
+msgstr "Chặn tài khoản"
+
+#: src/view/screens/ProfileList.tsx:747
+msgid "Block list"
+msgstr "Chặn danh sách"
+
+#: src/view/screens/ProfileList.tsx:742
+msgid "Block these accounts?"
+msgstr "Chặn các tài khoản này?"
+
+#: src/view/com/util/post-embeds/QuoteEmbed.tsx:83
+msgid "Blocked"
+msgstr "Đã bị chặn"
+
+#: src/screens/Moderation/index.tsx:274
+msgid "Blocked accounts"
+msgstr "Tài khoản đã bị chặn"
+
+#: src/Navigation.tsx:153
+#: src/view/screens/ModerationBlockedAccounts.tsx:108
+msgid "Blocked Accounts"
+msgstr "Tài khoản đã bị chặn"
+
+#: src/view/com/profile/ProfileMenu.tsx:336
+msgid "Blocked accounts cannot reply in your threads, mention you, or otherwise interact with you."
+msgstr "Tài khoản bị chặn không thể trả lời bài viết của bạn, đề cập đến bạn hoặc tương tác với bạn."
+
+#: src/view/screens/ModerationBlockedAccounts.tsx:116
+msgid "Blocked accounts cannot reply in your threads, mention you, or otherwise interact with you. You will not see their content and they will be prevented from seeing yours."
+msgstr "Tài khoản bị chặn không thể trả lời bài viết của bạn, đề cập đến bạn hoặc tương tác với bạn. Bạn sẽ không thể xem nội dung của họ và họ sẽ không được phép xem nội dung của bạn."
+
+#: src/view/com/post-thread/PostThread.tsx:412
+msgid "Blocked post."
+msgstr "Bài viết đã bị chặn."
+
+#: src/screens/Profile/Sections/Labels.tsx:173
+msgid "Blocking does not prevent this labeler from placing labels on your account."
+msgstr "Việc chặn không ngăn người dán nhãn này đặt nhãn trên tài khoản của bạn."
```

Comment:
Thanks, thought a bit about this but not sure if it makes any difference as there is little reason to emphasize "labeler" here.

Reply:
Thank for your input! I think emphasizing 'labeler' helps clarify the context or draw attention to its role in moderation scenario. But I'm open to suggestions if you see another way to approach it.

Reply:
Yeah. The concept of [label and labeler](https://docs.bsky.app/docs/advanced-guides/moderation#labelers) are indeed hard to understand in themselves.

Linguistically, "labeler" in this sentence is not a [proper noun](https://universaldependencies.org/u/pos/all.html#al-u-pos/PROPN) (danh từ riêng), so capitalizing it would falsely imply that it has a certain significance, while in this particular instance it is just referring to the profile the current user is viewing (see `src/screens/Profile/Sections/Labels.tsx` for the UI context). If we capitalize it here, however, we will have to do the same for all other places where the word is used, to keep consistency, and that might not be good because then it seems like we would be referring a one-and-only "Labeler".

If we want to remove the ambiguity around the word "labeler" here, what we can do is remove it altogether:

> Việc chặn không ngăn người này đặt nhãn trên tài khoản của bạn

as it will still convey the same meaning, referring to the profile being viewed.

---

Personally, I think is that users (both Vietnamese and non-Vietnamese speakers) will be confused in their first encounters with the concept of label & labeler regardless of location in the app, as they are not introduced in the onboarding process at all. And as they get on with the app and become more familiar with its features, they will gradually understand its lingo, and once that happen, these wording will start to make sense. In order words, in this instance, if a user already understands what label / labeler is, they will naturally understand what is written here; on the hand, if they don't know what label / labeler is there is no phrasing here that can help them at all.

Reply:
Thank you for your detailed explanation and thoughtful analysis! I agree with your points about the linguistic implications of capitalizing "labeler" and the potential inconsistency it could introduce across the app.